python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -370,16 +370,17 @@ def makeMinimalSchema(filterNameList, *, addCentroid=False,
 
             - If 2 or 3: add fields "coord_raErr" and "coord_decErr".
             - If 3: also add field "coord_radecErr".
-
-            Ignored if ``fullCoordinateCovariance=True``.
+                Ignored if ``fullCoordinateCovariance=True``.
```

Comment:
No, this line has an extra indent because it only applies to "If 3...". 
I don't think your second comment is correct. The only column that is in common is the `coord_ra`/`coord_dec` covariance. If `fullCoordinateCovariance` is True, then the `diagonalOnly` option at line 454 will be True, so the covariance will not be added there. You are correct that the two paths to adding ra/dec covariance end up with different names, but I don't see a way to avoid this.

Reply:
Sorry, my mistake. This is not the first time that I'd gotten confused about what `diagonalOnly` does, and I missed this one when I was cleaning up.

As for the indent, if it's supposed to be a continuation of "If 3", [then it should probably align with the "I"](https://developer.lsst.io/restructuredtext/style.html#lists). Though I'll admit I haven't checked what the built docs look like.